cita-auth/Cargo.toml

```diff
@@ -21,13 +21,11 @@ libproto = { git = "https://github.com/cryptape/cita-common.git", branch = "deve
 cita-crypto = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
 tx_pool = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
 jsonrpc-types = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
-core = { path = "../cita-chain/core" }
 uuid = { version = "0.7", features = ["v4"] }
 lru = "0.1"
 rayon = "1.0"
 hashable = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
-db = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
-evm = {path = "../cita-executor/evm"}
```

Comment:
When i try to remove db, the evm imports the cita-common. so it's conflict with rockstdb.
Now it's not a problem.